lib/cisco_node_utils/bgp.rb

```diff
@@ -119,32 +98,15 @@ def router_bgp(asnum, vrf, state='')
       set_args_keys_default
     end
 
-    def enable_create_router_bgp(asnum, vrf)
-      RouterBgp.enable
-      router_bgp(asnum, vrf)
-    end
-
     # Create one router bgp instance
     def create
-      if RouterBgp.enabled
-        router_bgp(@asnum, @vrf)
-      else
-        enable_create_router_bgp(@asnum, @vrf)
-      end
+      Feature.bgp_enable unless Feature.bgp_enabled?
```

Comment:
@glennmatthews Just had a thought about these  `unless _enabled?` checks. I'm thinking of just removing them from the callers and add the unless checks to the `_enable` methods themselves.  Is it worth changing?